pennylane/transforms/decompositions/clifford_t_transform.py

```diff
@@ -330,12 +331,71 @@ def _merge_param_gates(operations, merge_ops=None):
     return merged_ops, number_ops
 
 
-# pylint: disable=too-many-branches
+@lru_cache(maxsize=2000)
+def _map_wires(op, wire):
+    """Maps the operator to the provided wire."""
+    return op.map_wires({0: wire})
+
+
+class _CachedCallable:
+    """A class to cache the decomposition of the operators.
+
+    Args:
+        method (str): The method to be used for decomposition.
+        epsilon (float): The maximum permissible operator norm error for the decomposition.
+        cache_size (int): The size of the cache built for the decomposition function based on the angle.
+        **method_kwargs: Keyword argument to pass options for the ``method`` used for decompositions.
```

Comment:
Is this the standard way to repr. `method_kwargs` as `**method_kwargs` in docstrings?

Reply:
Afaik yeah. We use the same in the main transform and the sphinx docstring example [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html?highlight=module_level_function) also uses this.